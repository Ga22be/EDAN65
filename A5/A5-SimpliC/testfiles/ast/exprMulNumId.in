int mul() {
    int a;
    int b;
    b = 7;
    a = 9 * b;
}