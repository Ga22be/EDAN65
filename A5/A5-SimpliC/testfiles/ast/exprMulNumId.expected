Program
  List
    Function
      IdDecl ID="mul"
        IntType
      List
      List
        Declaration
          IdDecl ID="a"
            IntType
        Declaration
          IdDecl ID="b"
            IntType
        Assignment
          IdUse ID="b"
          Numeral NUMERAL="7"
        Assignment
          IdUse ID="a"
          Mul
            Numeral NUMERAL="9"
            IdUse ID="b"
